gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -23,6 +26,11 @@ import (
 	"go.opentelemetry.io/otel/metric"
 )
 
+var (
+	ErrInvalidRealm     = errors.New("invalid realm")
```

Comment:
It's not realm but package path.

And I think we already have the same regexp somewhere else, and potentially, and associated `Err`. Please recycle :)